### PR TITLE
Update snapshot-configs.md

### PR DESCRIPTION
**Pull request by quelhasu**
## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Review by KiraFuruichi — APPROVED** [review]
Great catch, thanks for opening up this PR! Feel free to merge this in when you have a chance. Please ensure this branch is up to date with current before merging in!

**Comment by mirnawong1** [comment]
thanks for opening this pr @quelhasu  and it looks like it's been a few months since my teammate Kira approved this. I'll go ahead and approve this on my end and looking forward to your next contribution. Have a great day!

**Pull request merged**
